tracer/src/Datadog.Trace/Ci/Test.cs

```diff
@@ -369,10 +379,28 @@ public void Close(TestStatus status, TimeSpan? duration, string? skipReason)
                 break;
         }
 
+        if (tags.Unskippable == "true" || tags.Unskippable?.ToLowerInvariant() == "true")
+        {
+            TelemetryFactory.Metrics.RecordCountCIVisibilityITRUnskippable(MetricTags.CIVisibilityTestingEventType.Test);
+        }
+
+        if (tags.ForcedRun == "true" || tags.ForcedRun?.ToLowerInvariant() == "true")
+        {
+            TelemetryFactory.Metrics.RecordCountCIVisibilityITRForcedRun(MetricTags.CIVisibilityTestingEventType.Test);
+        }
```

Comment:
Would this be better?
```suggestion
        if (tags.Unskippable is not null && string.Equals(tags.Unskippable, "true", StringComparison.OrdinalIgnoreCase))
        {
            TelemetryFactory.Metrics.RecordCountCIVisibilityITRUnskippable(MetricTags.CIVisibilityTestingEventType.Test);
        }

        if (tags.ForcedRun is not null && string.Equals(tags.ForcedRun, "true", StringComparison.OrdinalIgnoreCase))        {
            TelemetryFactory.Metrics.RecordCountCIVisibilityITRForcedRun(MetricTags.CIVisibilityTestingEventType.Test);
        }
```

Reply:
done in [c059cce](https://github.com/DataDog/dd-trace-dotnet/pull/4596/commits/c059cce9a1aca06218cc079d6a46a46c794397f3)